presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -877,6 +879,21 @@ private static String typedNull(String prestoType)
         return format("CAST(NULL AS %s)", prestoType);
     }
 
+    private static List<ParameterInfo> getDescribeInputParameterInfoList(ResultSet resultSet)
+            throws SQLException
+    {
+        ImmutableList.Builder<ParameterInfo> list = ImmutableList.builder();
+        while (resultSet.next()) {
+            String position = resultSet.getString("Position");
```

Comment:
I think the position is not required for us. Instead what if we directly maintain `ColumnInfo` ? It gives us precision, scale, signed details for a particular column. Can we create something like this
```
            ColumnInfo.Builder builder = new ColumnInfo.Builder()
                    .setCatalogName("") 
                    .setSchemaName("") 
                    .setTableName("") 
                    .setColumnLabel(resultSet.getString("Position"))
                    .setColumnName(resultSet.getString("Position")) 
                    .setColumnTypeSignature(column.getTypeSignature())
                    .setNullable(Nullable.UNKNOWN)
            setTypeInfo(builder, column.getTypeSignature());
            list.add(builder.build());
```

We just need to map the APIs for `ColumnInfo` when iterating over `PrestoParameterMetaData`. 

@electrum Your feedback on this approach ?

Reply:
That sounds like a good approach. It will allow us to add those additional things later if needed.